INDEX

```diff
@@ -23,6 +23,22 @@
     "official": false,
     "primary_pkg": "drupal8"
   },
+  "duplicati": {
+    "MANIFEST": "duplicati.json",
+    "name": "Duplicati",
+    "description": "Duplicati 2.0 Store securely encrypted backups on cloud storage services!",
+    "official": false,
+    "icon": "https://icons.freenas.org/community-icons/duplicati.png",
+    "primary_pkg": "",
+    "category": "backup"
+  },
+  "esphome": {
+        "MANIFEST": "esphome.json",
+        "name": "ESPHome",
+        "icon": "https://raw.githubusercontent.com/tprelog/freenas-plugin-index/11.3-RELEASE/icons/esphome.png",
```

Comment:
We need a `primary_pkg` key here, it could be null if there's no primary pkg. It is to keep consistent with all entries.